app/imports/ui/components/StudentCard.jsx

```diff
@@ -30,9 +30,9 @@ export default function StudentCard(props) {
           </div>
         <Card.Content>
           <Card.Meta>
-            <span >{email}</span>
+            <a href={'mailto:' + owner + '?Subject=Hello'} target="_top" id="blue-a-hyperlink" >{owner}</a>
```

Comment:
wut

Reply:
This links the email.  So when clicked on it'll launch the person's local email app and create a draft to be sent to the email that was clicked on. 
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_mailto